docs/docker.md

```diff
@@ -2,55 +2,103 @@
 
 # Docker
 
-This page list various operations that can be automated _via_ Docker.
+Production and development images for [Cozy Stack](https://cozy.io), Dockerfiles, docker-compose.yml & co lives [there](https://github.com/cozy/cozy-stack/tree/master/scripts)
 
-## Running a CouchDB instance
+## Requirements
 
-This will run a new instance of CouchDB in `single` mode (no cluster) and in
-`admin-party-mode` (no user). This command exposes couchdb on the port `5984`.
+You need docker and using docker-compose is a good idea. Here are the versions used for develoment
 
-```bash
-$ docker run -d \
-    --name cozy-stack-couch \
-    -p 5984:5984 \
-    -v $HOME/.cozy-stack-couch:/opt/couchdb/data \
-    apache/couchdb:2.3
-$ curl -X PUT http://127.0.0.1:5984/{_users,_replicator}
 ```
+➜  docker git:(master) docker -v
+Docker version 18.09.5, build e8ff056
+➜  docker git:(master) docker-compose version
+docker-compose version 1.24.0, build 0aa5906
+docker-py version: 3.7.2
+CPython version: 3.6.7
+OpenSSL version: OpenSSL 1.1.1  11 Sep 2018
+```
+
+Cozy Stack needs at least a CouchDB 2.3 to run.
+
+A reverse proxy is strongly recommended for HTTPS, Caddy being a good pick since it's able to provide on-demand TLS. Though any other can do.
+
+Redis for caching is optional.
+
+## Supported tags
+
+- `development`. Running `cozy-stack` compiled against `master` branch in `development` mode, a.k.a without enforcing HTTPS and other security features, so it can be run localy without certificates. Plus `couchdb` and `mailhog`, so it's a all in one environment for developers.
 
-Verify your installation at: http://127.0.0.1:5984/_utils/#verifyinstall.
+- `latest`. Running `cozy-stack` only compiled against `master` branch in `production` mode. *Use this one*.
 
-Note: for running some unit tests, you will need to use `--net=host` instead of
-`-p 5984:5984` as we are using CouchDB replications and CouchDB will need to be
-able to open a connexion to the stack.
+- `<version>` tag. Each one match a source code tag and are compiled in production mode, enforcing HTTPS and so on. Use this one.
+- `latest` tag. It matches the `master` branch of the source code, also compiled in production mode. Can be unstable, but you'll have the latest features here.
+- `development`. It also matches the `master` branch of the source code, but it's compiled in development mode, hence not enforcing HTTPS so it can be used localy without settings up
```

Comment:
`development` is listed twice, in first and last positions

Reply:
Latest too. Fixed.